packages/x-components/tests/e2e/common/common-steps.spec.ts

```diff
@@ -1,6 +1,6 @@
-import { PageableRequest } from '@empathyco/x-adapter';
 import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
-import '../cucumber/global-definitions';
+import { PageableRequest } from '../../../../search-adapter/types/index';
```

Comment:
Probably webstorm autoimport messed this import?
```suggestion
import { PageableRequest } from '@empathyco/x-adapter';
```